README.md

```diff
@@ -1,11 +1,263 @@
 # Social App Template
 
-Social Application Template that uses Gateway Services
+The Social Application Template (SAT) is an example client app that demonstrates how to use [Gateway Services](https://github.com/AmplicaLabs/gateway).
 
-## Frontend
+<!-- TABLE OF CONTENTS -->
 
-See [Frontend Readme](./frontend/README.md)
+# 📗 Table of Contents
 
-## Backend
+- [📖 About the Project](#about-project)
+- [🔍 Arch Map](#-arch-maps)
+- [🛠 Built With](#-built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live OpenAPI Docs](#-live-docs)
+- [💻 Getting Started](#-getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Deployment](#deployment)
+- [🤝 Contributing](#-contributing)
+- [❓FAQ](#faq)
+- [📝 License](#-license)
 
-See [Backend Readme](./backend/README.md)
+<!-- PROJECT DESCRIPTION -->The Social Application Template (SAT) is an example client app that demonstrates how to use [Gateway Services](https://github.com/AmplicaLabs/gateway).
+
+# 📖 Social Application Template (SAT) <a name="about-project"></a>
+
+The [Gateway Services](https://github.com/AmplicaLabs/gateway) are a suite of services designed to make interacting with [Frequency](https://github.com/frequency-chain/frequency) easy for applications integrating with [DSNP](https://dsnp.org/).
+The SAT is an example client that shows you how to use Gateway.
+
+Gateway enables web2 companies to use a simple gateway into web3 using tooling they are accustomed to. It aims to let service providers self-serve with minimal help outside of docs and tools, rather than
+interacting with Frequency directly. Providers no longer need to fully understand blockchain tooling to build on DSNP over Frequency.
+
+<!-- Mermaid Arch maps -->
+
+## 🔭 Arch Maps
+
+### Overview of the Social App Template in relation to Gateway Services.
```

Comment:
In the diagram, "Webhook Handlers" as a common component should not be part of the Gateway Services, but part of SAT. (Individual services may have webhooks that invoke the Webhook Handlers in the SAT). Otherwise it's not clear that the webhooks are a way for Gateway Services to communicate OOB/async info back to SAT.